automatic/bleachbit.install/tools/chocolateyInstall.ps1

```diff
@@ -0,0 +1,10 @@
+﻿$toolsDir = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
+
+$packageArgs = @{
+packageName   = $env:ChocolateyPackageName
+file          = "$toolsDir\BleachBit-1.17-setup.exe"
+fileType      = 'exe'
+silentArgs    = "/S"
+}
```

Comment:
as per [1.1.8 Set `softwareName`][1.1.8], please do add the softwareName.

[1.1.8]: https://github.com/chocolatey/chocolatey-coreteampackages/blob/master/CONTRIBUTING.md#118-set-softwarename